ext/fs/in_memory_fs.rs

```diff
@@ -0,0 +1,424 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+#![allow(clippy::disallowed_types)]
```

Comment:
Are there many of them? Maybe leave a comment about this directive usage.

Reply:
I'm going to add a comment on this. It's just to use `Arc`, which is fine for this module.